### PR TITLE
14061: Fix blank screen in Edge browser

### PR DESCRIPTION
**Pull request by viswanadh-kotha**
Hi Team,

As part of this PR we have included the changes for adding polyfills in order to support IE edge browser in windows OS.

### CODE REVIEW

File: src/webui/src/polyfills.js [hunk 1]
```diff
@@ -146,3 +146,43 @@ if (!Array.prototype.find) {
         writable: true,
     });
 }
+
```
Comment (kyleestes) [inline src/webui/src/polyfills.js:149]: Was this code copied from some source? If so, let's put a link to the source as a comment.

Comment (tgunter-analyst) [reply src/webui/src/polyfills.js:149]: Good catch @kyleestes!

Comment (viswanadh-kotha) [reply src/webui/src/polyfills.js:149]: Yes Kyle, Sometimes we are facing issue with repeat function, so inorder to handle that error  we have taken reference from the below link 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Comment (viswanadh-kotha) [reply src/webui/src/polyfills.js:149]: Yes Kyle, Sometimes we are facing issue with repeat function, so inorder to handle that error  we have taken reference from the below link 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Comment (viswanadh-kotha) [reply src/webui/src/polyfills.js:149]: Yes Kyle, Sometimes we are facing issue with repeat function, so inorder to handle that error  we have taken reference from the below link 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

Comment (viswanadh-kotha) [reply src/webui/src/polyfills.js:149]: Yes Kyle, Sometimes we are facing issue with repeat function, so inorder to handle that error  we have taken reference from the below link 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

**Pull request merged**